tool/add_to_firebase.py

```diff
@@ -0,0 +1,114 @@
+# Temporary tool that reads from JSON and uploads it to Firebase
+# Relies on an undocumented .collections() API call and a hard coded list of top level collections
+
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import firestore
+import hashlib
+from core_data_modules.logging import Logger
+import time
+
+import json
+import sys
+
+log = Logger(__name__)
+
+firebase_client = None
+
+def init(CRYPTO_TOKEN_PATH):
+    global firebase_client
+    log.info("Setting up Firebase client")
+    firebase_cred = credentials.Certificate(CRYPTO_TOKEN_PATH)
+    firebase_admin.initialize_app(firebase_cred)
+    firebase_client = firestore.client()
+    log.info("Done")
+
+def push_collection_to_firestore(collection_root, documents):
```

Comment:
Please add issue to refactor out to helper library

Reply:
issue created, thanks